### PR TITLE
[Enhancement] Support minus output feature index in mobilenet_v3

### PR DESCRIPTION
**Pull request by luminxu**
## Motivation

Fix typo in mobilenet_v3 backbone

## Modification

Modify `$mmpose/models/backbones/mobilenet_v3.py` to delete the reduplicate initialization and allow ``-1'' to indicates output tensors from final stage.

## Checklist

**Before PR**:

- [x] I have read and followed the workflow indicated in the [CONTRIBUTING.md](https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md) to create this PR.
- [x] Pre-commit or linting tools indicated in [CONTRIBUTING.md](https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md) are used to fix the potential lint issues.
- [x] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [x] New functionalities are covered by complete unit tests. If not, please add more unit tests to ensure correctness.
- [x] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [x] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: mmpose/models/backbones/mobilenet_v3.py [hunk 1]
```diff
@@ -164,6 +162,8 @@ def forward(self, x):
             x = layer(x)
             if i in self.out_indices:
                 outs.append(x)
+            elif i == (len(self.layers) - 1) and -1 in self.out_indices:
```
Comment (ly015) [inline mmpose/models/backbones/mobilenet_v3.py:165]: What about supporting general negative index, like -1, -2, ...?

Comment (luminxu) [reply mmpose/models/backbones/mobilenet_v3.py:165]: It is good. I will do it.

File: mmpose/models/backbones/mobilenet_v3.py [hunk 2]
```diff
@@ -86,8 +86,6 @@ def __init__(self,
             raise ValueError('frozen_stages must be in range(-1, '
                              f'{len(self.arch_settings[arch])}). '
                              f'But received {frozen_stages}')
-        self.out_indices = out_indices
-        self.frozen_stages = frozen_stages
```
Comment (jin-s13) [inline mmpose/models/backbones/mobilenet_v3.py:90]: why these two lines are deleted?

Comment (luminxu) [reply mmpose/models/backbones/mobilenet_v3.py:90]: They are reduplicate. Line 94 & 95 are the same in the original code .

**Review by jin-s13 — APPROVED** [review]

**Review by ly015 — APPROVED** [review]

**Pull request merged**